### PR TITLE
refactor(client): rename `Client` to `Prisma`

### LINKED ISSUE #205: Rename `Client` to `Prisma`
## Problem

When using Prisma alongside other frameworks, it is not immediately clear what `Client` is a reference to - is it a HTTP Client? An external API Client?

## Suggested solution

We should rename the `Client` to `Prisma`, this makes it very clear. However we should also keep the `Client` name for backwards compatibility.

```py
class Prisma:
    ...

Client = Prisma
```

We should also update the documentation accordingly.

## Additional context

There is one downside to this, the python convention is to instantiate classes with the lowercase class name, e.g.

```py
prisma = Prisma()
```

As the package is also named `prisma` there could be name conflicts that arise from this.

### PR DESCRIPTION
**Pull request by kivo360**
## Change Summary

Closes #205

## Checklist

- [x] Unit tests for the changes exist
- [x] Tests pass without significant drop in coverage
- [x] Documentation reflects changes where applicable
- [x] Test snapshots have been [updated](https://prisma-client-py.readthedocs.io/en/latest/contributing/contributing/#snapshot-tests) if applicable

## Agreement

By submitting this pull request, I confirm that you can use, modify, copy and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by kivo360** [comment]
Mind merging this in?

File: src/prisma/utils.py [hunk 1]
```diff
@@ -17,6 +17,7 @@ def _env_bool(key: str) -> bool:
 
 DEBUG = _env_bool('PRISMA_PY_DEBUG')
 DEBUG_GENERATOR = _env_bool('PRISMA_PY_DEBUG_GENERATOR')
+MAX_FN_SIGNATURE = 90
```
Comment (RobertCraigie) [inline src/prisma/utils.py:20]:
```suggestion
```

File: src/prisma/generator/templates/client.py.jinja [hunk 2]
```diff
@@ -502,3 +503,5 @@ class {{ model.name }}BatchActions:
 
 
 {% endfor %}
+
+Client = Prisma
```
Comment (RobertCraigie) [inline src/prisma/generator/templates/client.py.jinja:507]:
```suggestion
Client = Prisma

```

File: src/prisma/generator/templates/client.py.jinja [hunk 3]
```diff
@@ -3,7 +3,7 @@
 # -- template client.py.jinja --
 from types import TracebackType
 
-from . import types, models, errors, actions
+from . import types, models, errors, actions, utils
```
Comment (RobertCraigie) [inline src/prisma/generator/templates/client.py.jinja:6]:
```suggestion
from . import types, models, errors, actions
```

File: mypy.ini [hunk 4]
```diff
@@ -21,3 +21,4 @@ disallow_untyped_calls = True
 disallow_subclassing_any = True
 disallow_incomplete_defs = True
 disallow_untyped_decorators = True
+cache_fine_grained=True
```
Comment (RobertCraigie) [inline mypy.ini:24]:
```suggestion
cache_fine_grained = True
```

File: src/prisma/__init__.py [hunk 5]
```diff
@@ -6,6 +6,7 @@
 __copyright__ = 'Copyright 2020-2021 RobertCraigie'
 __version__ = '0.6.0a'
 
+
```
Comment (RobertCraigie) [inline src/prisma/__init__.py:9]:
```suggestion
```

File: src/prisma/__init__.py [hunk 6]
```diff
@@ -18,7 +19,10 @@
         models as models,
         partials as partials,
         types as types,
+        utils as utils,
     )
+
+
```
Comment (RobertCraigie) [inline src/prisma/__init__.py:25]:
```suggestion
    )
```

**Review by RobertCraigie — CHANGES_REQUESTED** [review]
Just a lot of minor nitpicks :)

You'll have to update the snapshots again unfortunately

**Comment by kivo360** [comment]
What happened here? I just saw the tests didn't pass.

**Comment by RobertCraigie** [comment]
@kivo360 I don't know, I've made some refactoring changes so that the `http.py` file is only available post-generation. Try merging main into your branch and see if it fixes the tests.

**Review by RobertCraigie — APPROVED** [review]

**Pull request merged**